debug.log

```diff
@@ -0,0 +1 @@
+[0220/151449.180:ERROR:crash_report_database_win.cc(428)] unexpected header
```

Comment:
this is supposed to be ignored, so maybe this was already tracked, tri to untrack it or delete from the repo